### ISSUE TITLE
Isue with aws_eks_node_group_group documentation

### ISSUE THREAD

**Issue by paul-lupu** [hashicorp/terraform-provider-aws#26535]
TL;DR: 
Update the following on the example listed here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eks_node_group
"""
  update_config {
    max_unavailable = 2 => 1
  }
"""
Hello, for the tutorial listed here, trying to use the examples leads to the following error

 Error: error creating EKS Node Group (mi-services-1-22-6:managed): InvalidParameterException: max-unavailable must be within the range 1 and max size of the node group
│ {
│   RespMetadata: {
│     StatusCode: 400,
│     RequestID: "e94bf956-e47d-4d14-be0a-398c7879c218"
│   },
│   ClusterName: "mi-services-1-22-6",
│   Message_: "max-unavailable must be within the range 1 and max size of the node group",
│   NodegroupName: "managed"
│ }
│ 
│   with aws_eks_node_group.managed,
│   on eks-v1-22-6.tf line 1, in resource "aws_eks_node_group" "managed":
│    1: resource "aws_eks_node_group" "managed" {

The following works: 
"""
  update_config {
    max_unavailable = 1
  }
"""

**Comment by justinretzolk** [comment]
Hey @paul-lupu 👋 Thank you for taking the time to raise this! I've opened a PR to correct the example.
